docs/updating/4_update_2.5.md

```diff
@@ -104,14 +104,25 @@ bin/console cache:pool:clear cache.redis
     CREATE INDEX ezpage_pages_content_id_version_no ON ezpage_pages(content_id, version_no);
     ```
 
-## Updating to 2.5.6
+## Updating to 2.5.15
 
-### Password expiration
+### Powered By Header
 
-Run the following script to update the database:
+In order to be able to promote use of eZ Platform, `ezsystems/ez-support-tools` v1.0.9 adds capability to set powered by header.
```

Comment:
```suggestion
In order to promote use of eZ Platform, `ezsystems/ez-support-tools` v1.0.9 sets the Powered-By header.
```
Or maybe, as we shouldn't expect users know which package has which version in which release:
```suggestion
In order to promote use of eZ Platform, `ezsystems/ez-support-tools` as of eZ Platform v2.5.15 sets the Powered-By header.
```